cuda/sample_consensus/src/ransac.cu

```diff
@@ -54,14 +55,14 @@ namespace pcl
     RandomSampleConsensus<Storage>::computeModel (int debug_verbosity_level)
     {
       // Warn and exit if no threshold was set
-      if (threshold_ == DBL_MAX)
+      if (threshold_ == std::numeric_limits<double>::max())
       {
         std::cerr << "[pcl::cuda::RandomSampleConsensus::computeModel] No threshold set!" << std::endl;
         return (false);
       }
 
       iterations_ = 0;
-      int n_best_inliers_count = -INT_MAX;
+      int n_best_inliers_count = std::numeric_limits<int>::lowest();
```

Comment:
Just a comment on this: while `std::numeric_limits<int>::lowest()` resolves to `INT_MIN` (might be `-2147483648`, depending on the platform), which is not equal to `-INT_MAX` (might be `-2147483647`, depending on the platform), your change makes sense :+1: